tests/test_forum.py

```diff
@@ -213,15 +214,33 @@ async def test_close_and_reopen_forum_topic(self, bot, forum_group_id, real_topi
             chat_id=forum_group_id,
             message_thread_id=message_thread_id,
         )
-        assert result is True
+        assert result is True, "Failed to close forum topic"
         # bot will still be able to send a message to a closed topic, so can't test anything like
         # the inability to post to the topic
 
         result = await bot.reopen_forum_topic(
             chat_id=forum_group_id,
             message_thread_id=message_thread_id,
         )
-        assert result is True
+        assert result is True, "Failed to reopen forum topic"
+
+    # async def test_unpin_all_forum_topic_messages(self, bot, forum_group_id, real_topic):
```

Comment:
we can uncomment and use [xfail](https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-mark-xfail-ref) instead :)